FBTweak/FBTweaksDisplayUtils.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2018 Lightricks. All rights reserved.
+// Created by Yonatan Oren.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FBTweaksDisplayUtils : NSObject
+
+/// Convert a date to a short formatted string in the device's timezone.
++ (NSString *)getSimpleLocalFormattedDate:(NSDate *)date;
```

Comment:
This is not the best option. Either add a category on `NSDateFormatter` which returns the formatter (using a class property perhaps) and the implementation calls the converts using the formatter. Or you can have a category on `NSDate`.

Reply:
As discusses, remote `get` and let merge.